apps/src/blockly/eventHandlers.ts

```diff
@@ -117,3 +120,78 @@ export function reflowToolbox() {
     modalWorkspace?.getFlyout()?.reflow();
   }
 }
+export function updateBlockLimits(event: Abstract) {
+  // This check is to update bubbles that show block limits whenever
+  // blocks on the main workspace are updated.
+  if (
```

Comment:
nit: We could add an `isRelevantEvent` helper for this.

```
if (!isRelevantEvent(event)) {
  return;
}
```

Reply:
Would that help with readability? Could also do something like `![...expectedEventTypes].includes(event.type)`